docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -42,7 +42,7 @@ In this output you can see:
 * The names that you give the play and each task.
   You should always use descriptive names that make it easy to verify and troubleshoot playbooks.
 
-* The ``Gather Facts`` task runs implicitly.
+* The ``Gathering Facts`` task runs implicitly.
```

Comment:
```suggestion
* The "Gathering Facts" task runs implicitly.
```

Reply:
Perhaps we should use double quotes instead of the backticks? When I first looked at this change I thought the backticks meant it was referencing the `ansible.builtin.gather_facts` module.

I think maybe the double quotes makes it more clear that it's referencing the preceding sample output. What do you think @izeye ?

Reply:
@oraNod Thanks for the feedback! I agree with your suggestion and applied it in 6edaaa2.